presto-kudu/conf/docker-compose.yml

```diff
@@ -0,0 +1,118 @@
+        --webserver_doc_root=/opt/kudu/www
+        --logtostderr
+        --rpc_bind_addresses=0.0.0.0:7051
+        --rpc_advertised_addresses=${KUDU_CLUSTERIP}:7051
+        --webserver_port=8051
+        --webserver_advertised_addresses=${KUDU_CLUSTERIP}:8051
+        --use_hybrid_clock=false
+  kudu-master2:
+    image: apache/kudu:${KUDU_VERSION:-latest}
+    ports:
+      - "7151:7151"
+      - "8151:8151"
+    command: ["master"]
+    environment:
+      - KUDU_MASTERS=${KUDU_CLUSTERIP}:7051,${KUDU_CLUSTERIP}:7151,${KUDU_CLUSTERIP}:7251
+      - >
+        MASTER_ARGS=--fs_wal_dir=/var/lib/kudu/master
+        --webserver_doc_root=/opt/kudu/www
+        --logtostderr
+        --rpc_bind_addresses=0.0.0.0:7151
+        --rpc_advertised_addresses=${KUDU_CLUSTERIP}:7151
+        --webserver_port=8151
+        --webserver_advertised_addresses=${KUDU_CLUSTERIP}:8151
+        --use_hybrid_clock=false
+  kudu-master3:
+    image: apache/kudu:${KUDU_VERSION:-latest}
+    ports:
+      - "7251:7251"
+      - "8251:8251"
+    command: ["master"]
+    environment:
+      - KUDU_MASTERS=${KUDU_CLUSTERIP}:7051,${KUDU_CLUSTERIP}:7151,${KUDU_CLUSTERIP}:7251
+      - >
+        MASTER_ARGS=--fs_wal_dir=/var/lib/kudu/master
+        --webserver_doc_root=/opt/kudu/www
+        --logtostderr
+        --rpc_bind_addresses=0.0.0.0:7251
+        --rpc_advertised_addresses=${KUDU_CLUSTERIP}:7251
+        --webserver_port=8251
+        --webserver_advertised_addresses=${KUDU_CLUSTERIP}:8251
+        --use_hybrid_clock=false
+
+  kudu-tserver1:
+    image: apache/kudu:${KUDU_VERSION:-latest}
+    depends_on:
+      - kudu-master1
+      - kudu-master2
+      - kudu-master3
+    ports:
+      - "7050:7050"
+      - "8050:8050"
+    command: ["tserver"]
+    environment:
+      - KUDU_MASTERS=${KUDU_CLUSTERIP}:7051,${KUDU_CLUSTERIP}:7151,${KUDU_CLUSTERIP}:7251
+      - >
+        TSERVER_ARGS=--fs_wal_dir=/var/lib/kudu/tserver
+        --webserver_doc_root=/opt/kudu/www
+        --logtostderr
+        --rpc_bind_addresses=0.0.0.0:7050
+        --rpc_advertised_addresses=${KUDU_CLUSTERIP}:7050
+        --webserver_port=8050
+        --webserver_advertised_addresses=${KUDU_CLUSTERIP}:8050
+        --use_hybrid_clock=false
+  kudu-tserver2:
+    image: apache/kudu:${KUDU_VERSION:-latest}
+    depends_on:
+      - kudu-master1
+      - kudu-master2
+      - kudu-master3
+    ports:
+      - "7150:7150"
+      - "8150:8150"
+    command: ["tserver"]
+    environment:
+      - KUDU_MASTERS=${KUDU_CLUSTERIP}:7051,${KUDU_CLUSTERIP}:7151,${KUDU_CLUSTERIP}:7251
+      - >
+        TSERVER_ARGS=--fs_wal_dir=/var/lib/kudu/tserver
+        --webserver_doc_root=/opt/kudu/www
+        --logtostderr
+        --rpc_bind_addresses=0.0.0.0:7150
+        --rpc_advertised_addresses=${KUDU_CLUSTERIP}:7150
+        --webserver_port=8150
+        --webserver_advertised_addresses=${KUDU_CLUSTERIP}:8150
+        --use_hybrid_clock=false
+  kudu-tserver3:
```

Comment:
why you cannot no longer scale with docker compose?

Reply:
its giving problems with communication between cluster tservers.
and the original docker-compose dont have docker scale.